pkg/mcs/resource_manager/server/server.go

```diff
@@ -275,6 +279,9 @@ func (s *Server) startServer() error {
 
 	// Server has started.
 	atomic.StoreInt64(&s.isServing, 1)
+	// register to etcd
```

Comment:
do we also need to register when running legacy server?

Reply:
No, it will only register when resource manager is started as a independent binary.

Reply:
I think this comment is useless.

Reply:
I will remove it.